bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+
+K = D &times; FSR &div; FP<sub>P</sub><sup>K/D</sup> = D &times; FSR &times; FP<sub>P</sub><sup>D/K</sup>
+
+where
+
+- D: amount of the smart asset that is held
+- FSR: force-settlement ratio (e.g. 95%)
+- FP<sub>P</sub><sup>K/D</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of smart asset divided by collateral (e.g. 2 bitCNY per BTS)
+- FP<sub>P</sub><sup>D/K</sup>: published feed price 24 hours after the force settlement is initiated denominated in terms of collateral divided by smart asset (e.g. 0.5 BTS per bitCNY)
+
+When a smart asset is globally settled, it is still possible to force settle with two differences in comparison to normal operations:
+
+- force-settlement has no delay; and,
+- the price used is the global settlement price (P<sub>gs</sub><sup>K/D</sup>), instead of using the feed price 24 hours in the future, which is denominated in units of the smart asset divided by the collateral (e.g. 2 bitCNY per BTS).  This price is **held constant** until the smart asset is revived from global settlement.
+
+K = D &times; FSR &div; P<sub>gs</sub><sup>K/D</sup> = D &times; FSR &times; P<sub>gs</sub><sup>D/K</sup>
+
+A GS-protected smart asset can still force-settled just as in normal operations albeit at an amount of collateral that is related to the GSP price.
+
+K = D &times; FSR &div; (1.01 &times; P<sub>gs</sub><sup>K/D</sup>) = D &times; FSR &times; (P<sub>gs</sub><sup>D/K</sup> &div; 1.01)
+
+_P<sub>gs</sub><sup>K/D</sup> is **not held constant** and will vary as the margin called debt positions are [purchased on the open market](#gsp-activated-trading)_.
+
+|Comparison to Globally Settled Asset|
+|-|
+|_While GSP is active_, a holder of a smart asset will obtain the __same amount or more collateral__ by force-settling than if the asset had globally settled because the GSP price will ratchet towards the natural feed price (FP<sub>N</sub>) as margin calls are filled.|
+
+
+##### <div id="gsp-activated-reference-denominated-price" /> Reference-Denominated Price
+
+The price of a smart asset (e.g. bitCNY) with respect to its reference asset (e.g. fiat CNY) is always fluctuating wherever the smart asset is trading relative to its reference asset outside of the DEX (e.g. centralized exchanges, peer-to-peer exchanges, etc.).  Due to the [lower risk of borrowing new smart assets into existence while GSP is active](#lower-risk-for-new-debt), the supply of the smart asset could increase significantly.  If this increased supply reaches external markets without a matching demand, the reference-denominated price of the smart asset will fall.
+
+|Comparison to Globally Settled Asset|
+|-|
+|The reference-denominated price of the smart asset (e.g. the CNY-price of bitCNY) **might** decrease faster _than under global settlement_ **if** new supply of the smart asset is created during GSP.|
```

Comment:
The RDP(Reference-Denominated Price) is not only affected by supply, but also by overall collateral ratio and arbitrage opportunities. It's hard to tell if GSP brings more risks than GS.
* When GS is activated, if overall CR(collateral ratio) decrease due to price of collateral asset drops, RDP is more likely to drop at same time; if overall CR increase, RDP will raise. When GSP is activated, the price of collateral and price of bitAsset is not that corrective.
* When GSP is activated, if cheap bitAsset enters the market, there may be arbitrage opportunities to settle or sell them for collateral asset for profit, thus "RDP of the smart asset will fall" might be not always true.
* in case when (also usually) bitAsset is trading at a premium before GSP is activated, "RDP of the smart asset will fall" is more desired.

Reply:
@abitmore Please review the re-written section to see if adequately emphasizes these points